File: go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert returns string equivalent of a number
func Convert(number int) string {
	var result string = ""

	if number%3 == 0 {
		result += "Pling"
	}

	if number%5 == 0 {
		result += "Plang"
	}

	if number%7 == 0 {
		result += "Plong"
	}

	if result == "" {
		return strconv.Itoa(number)
	}

	return result
}

File: python/raindrops/raindrops.py
def convert(number):
    result = ""
    if not (number % 3):
        result += "Pling"

    if not (number % 5):
        result += "Plang"

    if not (number % 7):
        result += "Plong"

    return result if result else str(number)
